js/main.js

```diff
@@ -9,29 +9,94 @@
   All code in this file was authored by David Lin
 */
 
-var showSettings = function() {
-  $(".groupsContainer").hide();
-  $(".graphsContainer").hide();
-  $(".settingsContainer").show();
-}
-
-var showGroups = function() {
-  $(".settingsContainer").hide();
-  $(".graphsContainer").hide();
-  $(".groupsContainer").show();
-}
-
-var showGraphs = function() {
-  $(".groupsContainer").hide();
-  $(".settingsContainer").hide();
-  $(".graphsContainer").show();
-}
-
 $(function() {
   var directory = new air.File();
   var picturePath = "";
   var ratingsFile;
   var pictures = [];
+  var groups = [];
+
+  var loadPics = function() {
+    var domAdd = "";
+    pictures.forEach(function(picture) {
+      var path = new air.File(picture.filePath).url;
+      domAdd += '<div class="col-md-1"><img src="' + path + '" class="image"></div>';
+    });
+
+    // flush DOM then add current pictures
+    $('#unsorted').html('');
+    $('#unsorted').append(domAdd);
+
+    var groupCount = 1;
+    groups.forEach(function(group) {
+      var currentGroup = $("div[data-group='" + groupCount + "']");
```

Comment:
would it be cleaner here to just have each group associated by id, then fetch by id? Ex. `id='1'